Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Snacks.kt

```diff
@@ -100,7 +102,10 @@ fun SnackCollection(
                 modifier = Modifier.align(Alignment.CenterVertically)
             ) {
                 Icon(
-                    imageVector = Icons.Outlined.ArrowForward,
+                    imageVector = mirroringIcon(
```

Comment:
Could this use `mirroringBackIcon`?

Reply:
Nope, in this layout the arrow points the other way.

See the forward pointing arrows in the screenshot:
<img src="https://user-images.githubusercontent.com/6952116/119789724-9c06cb80-bed3-11eb-8382-63776de31d10.png" width="200px"/>